src/module.c

```diff
@@ -1977,6 +2014,23 @@ NAPI_METHOD(dcn_lot_get_timestamp) {
  * dc_msg_t
  */
 
+NAPI_METHOD(dcn_msg_get_parent) {
+  NAPI_ARGV(1);
+  NAPI_DC_MSG();
+
+  napi_value result;
+  dc_msg_t* msg = dc_msg_get_parent(dc_msg);
+
+  if (msg == NULL) {
+    NAPI_STATUS_THROWS(napi_get_null(env, &result));
```

Comment:
you really wanna throw here and not just return null/false/-1 whatever? We could always still throw in the higher bindings